### PR TITLE
Pulls coc copy changes from @Nandutu's branch

### LINKED ISSUE #209: Add emergency numbers to Code of Conduct page

### PR DESCRIPTION
**Pull request by jefftriplett**
These updates come from @Nandutu's #212 pull request and branch. 

Thank you @Nandutu for updating the copy them! 🎉 

Fixes #209

### CODE REVIEW

File: _pages/coc.html [hunk 1]
```diff
@@ -96,7 +96,23 @@ <h2 class="demp">
 
         <p>Conference staff will be happy to help participants contact venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.</p>
 
-        <p>If your physical safety or the physical safety of others is at risk, please dial 911.</p>
+        <p>+If your physical safety or the physical safety of others is at risk, please contact the relevant authorities below.</p>
```
Comment (williln) [inline _pages/coc.html]: There is an errant plus sign at the start of line 99

**Comment by williln** [comment]
LGTM once the plus sign is omitted. 

I might style the h3's as items in a `<ul>` personally? I think the separate headings are a little harder to read. I might also replace this: "Crime Check - To report Non-Emergency incidents:" with "Crime Check (to report non-emergency incidents):" for the same reason: explanatory text is harder to read when it's in title case). Those are personal preferences though; feel free to reject.

File: _pages/coc.html (hunk 1 above)
Comment (jefftriplett) [reply _pages/coc.html]: Grr, good catch. My bad on that one

**Comment by KatherineMichel** [comment]
Thank you everyone! @Nandutu @jefftriplett @williln

**Pull request merged**